packages/lesmis/package.json

```diff
@@ -31,8 +30,6 @@
     "@material-ui/icons": "^4.9.1",
     "@material-ui/lab": "^4.0.0-alpha.57",
     "@material-ui/styles": "^4.9.10",
-    "@tupaia/access-policy": "3.0.0",
```

Comment:
Yeah I removed this from a couple of places because it wasn't being used inside the package, so I'm assuming it was a copy paste of another package.json when these got spun up